### ISSUE TITLE
SQLite3 driver incompatibility

### ISSUE THREAD

**Issue by kvsari** [dogmatiq/example#145]
# SQLite3 driver incompatibility
I cloned the repo and ran;
```
make
```
Then ran
```
go run cmd/bank/main.go
```

I expected the example to run but instead got;
```
panic: sqlprojection.mysqlDriver is not compatible with *sqlite3.SQLiteDriver: unable to open database file: no such file or directory; sqlprojection.postgresDriver is not compatible with *sqlite3.SQLiteDriver: unable to open database file: no such file or directory; sqlprojection.sqliteDriver is not compatible with *sqlite3.SQLiteDriver: unable to open database file: no such file or directory; could not find a driver that is compatible with *sqlite3.SQLiteDriver

goroutine 1 [running]:
github.com/dogmatiq/example/database.MustNew(...)
        /home/stephan/builds/dogmatic/example/database/db.go:44
main.main()
        /home/stephan/builds/dogmatic/example/cmd/bank/main.go:19 +0x618
exit status 2
```

**Comment by kvsari** [comment]
Whoops, just needed to create the ./artifacts dir.

**Comment by jmalloc** [comment]
Thanks for the report! I've updated the example to automatically create this directory when run.

Fixed in 6d734f4
